### PR TITLE
Update Xamarin.Android.Support.CustomTabs and v7.AppCompat to version…

### PR DESCRIPTION
**Pull request by jennyf19**
… 27.0.2

Tested w/the Xamarin Android App

Addressing #520 and #553 

Due to issue w/Xamarin Forms, CustomTabs and AppCompat have to be manually updated in the respective csproj files, or we can keep on top of the updates.

The Activity was coming in null, so I used what we have in MSAL Core library and that fixed the problem.

### CODE REVIEW

File: src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj [hunk 1]
```diff
@@ -167,10 +167,10 @@
   <!-- CustomTabs and AppCompat must reference the same version -->
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
     <PackageReference Include="Xamarin.Android.Support.CustomTabs">
-      <Version>25.4.0.2</Version>
+      <Version>27.0.2</Version>
```
Comment (henrik-me) [inline src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj]: ><Version>27.0.2</Version> [](start = 6, length = 25)

Is this installed on the build servers?

**Comment by henrik-me** [comment]
Consider if the test apps should be updated to v8.0 or v8.1

File: src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj (hunk 1 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj]: I will check

**Comment by jennyf19** [comment]
v8.1 - 27.0.2 is only compatible w/8.1. having other dependency issues that i am working on resolving before merging this.

---
In reply to: [391163651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391163651) [](ancestors = 391163651)

**Comment by henrik-me** [comment]
ack

---
In reply to: [391164233](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391164233) [](ancestors = 391164233,391163651)

**Comment by jennyf19** [comment]
Tomorrow, would you mind pull in these changes and checking things work for you locally? Thanks

---
In reply to: [391164233](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391164233) [](ancestors = 391164233,391163651)

**Comment by henrik-me** [comment]
@jennyf19 : will try now and let you know

**Comment by henrik-me** [comment]
Building the core components and running utests works fine. I can't get the XForms app to build, VS claims there is a missing reference to MonoAndroid,v8.1 however when looking at the project file it points to the right file... not sure why this is.

**Comment by henrik-me** [comment]
Got the test app to work on a Pixel 2. and successfully acquired a token. Local builds seems to only have the same issues as MonoAndroid,v7... so at least as good as before :)

**Comment by henrik-me** [comment]
By this change we are potentially breaking some older apps? Also my device has 8.1 installed... do you have a v7 device to validate on as well?

**Comment by jennyf19** [comment]
i will check tomorrow. i think i have a v7 device. 

---
In reply to: [391193786](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391193786) [](ancestors = 391193786)

**Comment by jennyf19** [comment]
thanks!!

---
In reply to: [391193957](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391193957) [](ancestors = 391193957,391193786)

**Comment by jennyf19** [comment]
I tested on Plus One device w/V7.1.1 and was able to acquire token. Should I go ahead and merge this PR?

---
In reply to: [391193786](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/558#issuecomment-391193786) [](ancestors = 391193786)

**Pull request merged**